care/facility/models/patient_base.py

```diff
@@ -53,10 +53,17 @@ def reverse_choices(choices):
     "Diabetes": 2,
     "Heart Disease": 3,
     "HyperTension": 4,
-    "Kidney Diseases": 5,
-    "Lung Diseases/Asthma": 6,
-    "Cancer": 7,
-    "OTHER": 8,
+    "Chronic Renal Disease": 5,
+    "Asthma": 6,
+    "COPD": 7,
+    "Bronchitis": 8,
+    "Chronic Neurological Or Neuromuscular Disease": 9,
+    "Immunocompromised Condition": 10,
+    "Liver Disease": 11,
+    "TB": 12,
+    "Other Chronic Lung Diseases": 13,
+    "Cancer": 14,
+    "OTHER": 15,
```

Comment:
Migration of existing records would be incorrect this way. For example, if a patient had a disease "OTHER" selected, it'd be "Bronchitis" once this migration runs, which is incorrect. 

Keep the same numeric value for things that are only just renamed. All new diseases that are being introduced to be added with a new numeric value not used before.